src/uct/gaudi/api/Makefile.am

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (C) Intel Corporation, 2025. ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+if ENABLE_GAUDI_TOPO_API
+
+gaudiincludedir      = $(includedir)/uct/gaudi
```

Comment:
AFAIU this means gaudi_topo.h is installed. is it expected that applications will include gaudi_topo.h directly? since it's not the standard usage mode